src/main/java/com/minecolonies/entity/pathfinding/PathNavigate.java

```diff
@@ -74,18 +69,13 @@ public Path getPathToPos(BlockPos pos)
         return null;
     }
 
-    @Override
-    protected boolean isDirectPathBetweenPoints(final Vec3d Vec3d, final Vec3d Vec3d1, final int i, final int i1, final int i2)
-    {
-        //we don't use, so it doesn't matter
-        return false;
-    }
-
-    @Override
-    public void setBreakDoors(boolean canBreakDoors)
-    {
-        this.canBreakDoors = canBreakDoors;
-    }
+    //  Re-enable this if path shortcutting becomes a problem; then entities will move more rigidly along world grid
+//    @Override
+//    protected boolean isDirectPathBetweenPoints(final Vec3d Vec3d, final Vec3d Vec3d1, final int i, final int i1, final int i2)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)